playground/infrastructure/cloudbuild/cloudbuild_helm_update.yaml

```diff
@@ -0,0 +1,98 @@
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+steps:
+  - name: ubuntu
+    args:
+      - '-c'
+      - >-
+        apt update > /dev/null
+
+        apt-get install -y wget unzip software-properties-common git curl
+        apt-transport-https ca-certificates gnupg jq > /dev/null
+
+        echo "deb https://packages.cloud.google.com/apt cloud-sdk main" | tee -a
+        /etc/apt/sources.list.d/google-cloud-sdk.list
+
+        curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add
+        -
+
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+
+        add-apt-repository "deb [arch=amd64]
+        https://download.docker.com/linux/ubuntu focal stable" > /dev/null
+
+        apt install -y docker-ce > /dev/null
+
+        wget
+        https://releases.hashicorp.com/terraform/1.4.2/terraform_1.4.2_linux_amd64.zip 
+
+        unzip terraform_1.4.2_linux_amd64.zip
+
+        mv terraform /usr/local/bin/terraform
+
+        terraform --version
+
+        apt update -y > /dev/null && apt install -y google-cloud-sdk
+        openjdk-11-jdk > /dev/null 
+
+        git clone --branch merged_pg_gke_deployment
+        https://github.com/akvelon/beam.git
+
+        curl -fsSLo get_helm.sh
+        https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 >
+        /dev/null
+
+        chmod +x get_helm.sh && ./get_helm.sh > /dev/null
+
+        apt-get install google-cloud-sdk-gke-gcloud-auth-plugin
+
+        cd beam
+
+        mkdir playground/terraform/environment/myenv
+
+        gcloud compute scp --zone=us-central1-a
+        instance-1:/home/ruslan_ikhsanov/beam/playground/terraform/environment/myenv/terraform.tfvars
+        playground/terraform/environment/myenv/terraform.tfvars
+
+        gcloud compute scp --zone=us-central1-a
+        instance-1:/home/ruslan_ikhsanov/beam/playground/terraform/environment/myenv/state.tfbackend
+        playground/terraform/environment/myenv/state.tfbackend
+
+        cat playground/terraform/environment/myenv/terraform.tfvars
+
+        cat playground/terraform/environment/myenv/state.tfbackend
+
+        gcloud auth configure-docker us-east1-docker.pkg.dev
+
+        gcloud container clusters get-credentials --region us-east1-b
+        playground-gke --project cloudbuildtest-382210
+
+        ./gradlew playground:terraform:gkebackend -Pproject_environment=myenv
+        -Pdocker-tag=$_TAG_NAME -Pdns-name=$_DNS_NAME -Psdk-tag=$_SDK_TAG
```

Comment:
i don't quite understand how these multiline commands are supposed to work given that there is no escaping `\` of the line break?

Reply:
This is not Bash script. The yaml syntax is corrected by UI in GCP console. It automatically breaks command to lines. Command separated by empty lines

Reply:
Can we extract this into a `.sh` file so it would be easier to understand?

Reply:
It is simple as is. We have dediacted shell files for more complex cases (CI/CD)